tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestMajors.g.cs

```diff
@@ -524,22 +524,22 @@ public class PackageVersionsLatestMajors
                 new object[] { string.Empty },
 #else
 #if NET461
-                new object[] { "2.2.7" },
+                new object[] { "2.2.8" },
```

Comment:
There should be a corresponding change in _PackageVersionsGeneratorDefinitions.json_, otherwise these will get wiped out when we run `GeneratePackageVersions`

Reply:
I've updated the json on 17c72a5